app/_hub/kong-inc/opentelemetry/how-to/_dynatrace.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Set up Dynatrace with OpenTelemetry
+nav_title: Using Dynatrace
+minimum_version: 3.8.x
+---
+
+Set up the OpenTelemetry plugin to send logs and metrics to Dynatrace.
+
+## Prerequisites
+* Install the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/installation/)
+* {{site.base_gateway}} 3.8+
+
+## Configure {{site.base_gateway}}
+
+Set the following parameters in your [`kong.conf`](/gateway/latest/production/kong-conf/) file:
+
+```
+tracing_instrumentations = all
+tracing_sampling_rate = 1.0
+```
+
+## Configure the OpenTelemetry plugin
+
+Adjust the `{OPENTELEMETRY_COLLECTOR}` variable with your own Collector endpoint:
```

Comment:
```suggestion
Adjust the `{OPENTELEMETRY_COLLECTOR}` variable with your own collector endpoint:
```
Looks like it would be lowercase here 
https://opentelemetry.io/docs/collector/